html/browsers/history/the-history-interface/history_go_to_uri.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<title>history.go()</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+  var t = async_test(undefined, {timeout:5000});
```

Comment:
This test needs a name. Maybe "history.go() must not navigate to URLs given as its argument; instead it must treat them as 0"